### PR TITLE
Fix credentials logging

### PR DESCRIPTION
**Pull request by dhlavac**
Temporarily fix before https://github.com/RedHatQE/widgetastic.core/pull/227 will be merged.

### CODE REVIEW

**Review by JaurbanRH — APPROVED** [review]

File: testsuite/ui/views/common/login.py [hunk 1]
```diff
@@ -22,7 +31,9 @@ def do_login(self, name, password):
         :return DashboardView page object
         """
         self.username_field.fill(name)
+        LOGGER.disabled = True
         self.password_field.fill(password)
```
Comment (jsmolar) [inline testsuite/ui/views/common/login.py:35]: use `self.fill_passwd` method here

File: pytest.ini [hunk 2]
```diff
@@ -21,5 +21,6 @@ junit_logging=all
 junit_family=xunit2
 junit_log_passing_tests=0
 asyncio_mode=auto
+log_cli = true
```
Comment (jsmolar) [inline pytest.ini:24]: why do we need `log_cli`?

Comment (dhlavac) [reply pytest.ini:24]: Not necessary but logs with it provide much more info for investigation, at least for UI tests. @mganisin what do you think?

Comment (mganisin) [reply pytest.ini:24]: good for your local PYTEST_ADDOPTS, not that great as global default. Even if considered good, certainly should not be introduced here, but in separate MR.

**Review by jsmolar — DISMISSED** [review]

**Comment by mganisin** [comment]
@dhlavac this requires your care

**Comment by dhlavac** [comment]
blocked by https://issues.redhat.com/browse/IQE-2070

**Review by JaurbanRH — APPROVED** [review]

**Review by jsmolar — APPROVED** [review]

**Pull request merged**